packages/front-end/components/HealthTab/DimensionIssues.tsx

```diff
@@ -274,11 +284,43 @@ export const DimensionIssues = ({ dimensionData, variations }: Props) => {
             </div>
           </>
         ) : (
-          <div className="pt-4 px-4">
-            <i className="text-muted">
-              No experiment dimensions with pre-defined slices available
-            </i>
-          </div>
+          <>
+            <div className="pt-4 px-4">
+              <i className="text-muted">
+                {`No experiment dimensions ${
+                  healthTabConfigParams.exposureQueryDimensions.length > 0
+                    ? "with pre-defined slices available"
+                    : "available"
+                }`}
+              </i>
+            </div>
+            {healthTabConfigParams.exposureQueryDimensions.length ? (
+              <div className="pt-4 d-flex justify-content-center">
+                <div>
+                  <a
+                    href="#"
+                    className="btn btn-outline-primary"
+                    onClick={() => {
+                      track("Health Tab Onboarding Opened", {
+                        source: "dimension-issues",
+                      });
+                      setSetupModalOpen(true);
+                    }}
+                  >
+                    Configure experiment dimension slices
+                  </a>
+                </div>
+                {setupModalOpen ? (
+                  <HealthTabOnboardingModal
+                    open={setupModalOpen}
+                    close={() => setSetupModalOpen(false)}
+                    healthTabConfigParams={healthTabConfigParams}
+                    healthTabOnboardingPurpose={"dimensions"}
+                  />
+                ) : null}
```

Comment:
You shouldn't need to do a ternary with `setupModalOpen` since the `HealthTabOnboardingModal` has access to the `open` state and should hide if it's `false`

Reply:
I think leaving it in forces it to re-render if you click back into the modal which is good because it resets the modal state. I removed it on the main health onboarding CTA to test and it would instead remember the last step of the modal (which would be the exit modal), so I think we should leave the ternary.